core/src/components/picker-column/picker-column.tsx

```diff
@@ -46,23 +39,11 @@ export class PickerColumn implements ComponentInterface {
    */
   @Prop() disabled = false;
 
-  /**
-   * A list of options to be displayed in the picker
-   */
-  @Prop() items: PickerColumnItem[] = [];
-
   /**
    * The selected option in the picker.
    */
   @Prop({ mutable: true }) value?: string | number;
 
-  /**
-   * The color to use from your application's color palette.
-   * Default options are: `"primary"`, `"secondary"`, `"tertiary"`, `"success"`, `"warning"`, `"danger"`, `"light"`, `"medium"`, and `"dark"`.
-   * For more information on colors, see [theming](/docs/theming/basics).
-   */
-  @Prop({ reflect: true }) color?: Color = 'primary';
```

Comment:
We could have picker-column-option inherit the color from picker-column, but that behavior was not scoped in the design document. As a result, I removed this prop.